invenio_rdm_records/contrib/codemeta/custom_fields.py

```diff
@@ -55,6 +55,7 @@
         dict(
             field="code:codeRepository",
             ui_widget="Input",
+            is_identifier=True,
             props=dict(
                 label="Repository URL",
```

Comment:
```suggestion

            props=dict(
                label="Repository URL",
                is_identifier=True,
```

minor: I would add this inside `props` which looks like a more generic key/value pair store for config.